plugins/modules/source_control/bitbucket/bitbucket_pipeline_variable.py

```diff
@@ -119,17 +119,16 @@ def get_existing_pipeline_variable(module, bitbucket):
 
     The `value` key in dict is absent in case of secured variable.
     """
-    content = {
-        'next': BITBUCKET_API_ENDPOINTS['pipeline-variable-list'].format(
-            username=module.params['username'],
-            repo_slug=module.params['repository'],
-        )
-    }
-
+    variables_base_url = BITBUCKET_API_ENDPOINTS['pipeline-variable-list'].format(
+        username=module.params['username'],
+        repo_slug=module.params['repository'],
+    )
     # Look through the all response pages in search of variable we need
-    while 'next' in content:
+    page = 1
+    while page:
```

Comment:
```suggestion
    while True:
```
If you're creating an infinite loop, be honest about it :)

Reply:
😀